atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -231,7 +231,7 @@ def run(self):
                                         '", "'.join(sorted(unknown_states)))
 
         rebuild = is_rebuild(self.workflow)
-        success = not self.workflow.build_failed
+        success = not self.workflow.build_result.is_failed()
```

Comment:
This should probably be `self.workflow.build_process_failed` anyway -- it cares about whether the build as a whole (plugins etc) has failed, rather than specifically the build step.